sys/net/link_layer/ieee802154/submac.c

```diff
@@ -56,7 +83,7 @@ static int _perform_csma_ca(ieee802154_submac_t *submac)
         ieee802154_radio_request_set_trx_state(dev, IEEE802154_TRX_STATE_TX_ON);
         /* delay for an adequate random backoff period */
         uint32_t bp = (random_uint32() & submac->backoff_mask) *
-                      CSMA_SENDER_BACKOFF_PERIOD_UNIT_MS;
+                      ieee802154_get_unit_backoff_period(submac);
```

Comment:
I think you also want to replace the `ACK_TIMEOUT_US` constant now :wink:

Reply:
Added it on https://github.com/RIOT-OS/RIOT/pull/16475/commits/031c1bb2611d547ad48a29103994f4429eeca5fd